internal/gitsources/github/github.go

```diff
@@ -122,16 +122,29 @@ func New(opts Opts) (*Client, error) {
 	}
 	httpClient := &http.Client{Transport: &TokenTransport{token: opts.Token, rt: transport}}
 
+	isPublicGithub := false
+	// TODO(sgotti) improve detection of public github url (handle also trailing slash)
 	if opts.APIURL == GitHubAPIURL {
+		isPublicGithub = true
+	}
+
+	if !strings.HasSuffix(opts.APIURL, "/") {
+		opts.APIURL += "/"
+	}
+
+	if isPublicGithub {
 		opts.WebURL = GitHubWebURL
 	} else {
+		if !strings.HasSuffix(opts.APIURL, "/api/v3/") {
+			opts.APIURL += "api/v3/"
+		}
```

Comment:
Should be placed after line number 143 (this if closure `if opts.WebURL == "" {`)
WebURL should not be ended with "api/v3/"